### ISSUE TITLE
improvement in formatting

### ISSUE THREAD

**Issue by jorre127** [KalleHallden/exer_log#53]
Maybe it would be a good idea to run dart format for the whole project before making every pull request, this way the formatting is the same across the app. Currently the problem is that every contributor has different formatters, line lengths etc., which makes it harder when reviewing pull requests to see what are the actual changes, and what is just the formatting changing.

**Comment by rileyhawk1417** [comment]
Would it work just to use the default Formatter that comes with Flutter?
When I reinstall my dart plugins I get prompted to add a config file for the dart formatter.
Not sure if it's relevant though.

**Comment by jorre127** [comment]
Yeah but I don't think we can force everybody to use the same formatter. Also people might have different settings like line length and such. this is why in the pull request right now everybody's formatting looks different which makes it hard to see the changes. The dart format command should always format the same, so if that gets run before each PR we should only see the actual changes, not any formatting changing.

**Comment by rileyhawk1417** [comment]
I see so I guess probably having a uniform format or code structure should just work for everyone then?
For example writing functions and variables in a certain way that makes it clean and not congested.

**Comment by jorre127** [comment]
Yeah I good start would be to write commas at the end of every line and attribute, this forces the formatter to put these on separate lines and help to make the formatting more uniform.

**Comment by rileyhawk1417** [comment]
So I guess it's just a matter of writing a config file that will be read by the formatter, then format the code accordingly?

**Comment by jorre127** [comment]
Yeah or just use the dart format command, this automatically formats the code according to dart guidelines, so it will be the same for everyone.

**Comment by MhouneyLH** [comment]
If nobody has already started I would like to add this to the codebase. :)
We just need to add a new job in the workflows, that runs flutter format. Should not be that big of a deal, shouldn't it?
